base/server/upgrade/11.9.0/01-EnableEST.py

```diff
@@ -42,37 +46,38 @@ def upgrade_subsystem(self, instance, subsystem):
         self.add_sd_type(subsystem)
 
     def enable_est_profile(self, instance, subsystem):
-        estProfile = 'estServiceCert'
+        estProfiles = ['estServiceCert', 'estFullcmcDeviceCert']
         # Read the available profile list
         profile_list = subsystem.config.get('profile.list').split(',')
 
-        if estProfile in profile_list:
-            logger.info('EST profile already in use.')
-            return
+        for estProfile in estProfiles:
+            if estProfile in profile_list:
+                logger.info('EST profile %s already in use.', estProfile)
+                continue
 
```

Comment:
If estProfiles are configured the method should stop otherwise at every server restart the profiles will be overwritten with the version in share folder, removing any possible customisation.